### PR TITLE
[WIP] Add v1 styling from approved designs

### PR DESCRIPTION
**Pull request by surudhb**

### CODE REVIEW

File: frontend/src/components/Header.tsx [hunk 1]
```diff
@@ -20,35 +20,41 @@ export const Header: React.FC<Props> = () => {
...
-                <option key={vault.id} value={vault.id}>
-                  {' '}
-                  {displayAddr(vault.id)}{' '}
-                </option>
-              ))}
-            </Select>
-          </div>
+          <Dropdown
+            options={vaults.map((vault) => vault.id)}
+            selectedOption={vaults[0].id}
```
Comment (jx3yang) [inline frontend/src/components/Header.tsx]: We probably want `selectedVault ? selectedVault.id : vaults[0].id`

File: frontend/src/components/GeyserStakeView.tsx [hunk 2]
```diff
@@ -60,22 +44,23 @@ export const GeyserStakeView: React.FC<Props> = () => {
...
-          <div>New stake: {formatDisplayAmount(userStake.add(parsedAmount))}</div>
-        </>
-      )}
+    <GeyserStakeViewContainer>
+      <WalletBalance
+        parsedAmount={parsedAmount}
+        walletAmount={walletAmount}
+        decimals={stakingTokenDecimals}
+        symbol={stakingTokenSymbol}
+      />
```
Comment (jx3yang) [inline frontend/src/components/GeyserStakeView.tsx:53]: I think we will need to also change this to show wallet balance when staking and staked balance when unstaking right? We can fix it later, but maybe we can put a TODO somewhere

File: frontend/src/components/WalletBalance.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import styled from 'styled-components/macro'
+import tw from 'twin.macro'
+import { formatAmount } from '../utils/amount'
+
+interface Props {
+  parsedAmount: BigNumber
+  walletAmount: BigNumber
+  decimals: number
+  symbol: string
+}
```
Comment (jx3yang) [inline frontend/src/components/WalletBalance.tsx:11]: Quick qst: would it be better to use the wallet context here instead of getting those as props?

Comment (surudhb) [reply frontend/src/components/WalletBalance.tsx:11]: Both are good approaches imo. Went with passing in props for this one to generally make all the UI components as dumb as possible and leave the data handling to either the context or the parent component

**Review by jx3yang — APPROVED** [review]

**Pull request merged**